app/components/channel_list/categories/unreads.test.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Database} from '@nozbe/watermelondb';
+import React from 'react';
+
+import {renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import UnreadsCategory from './unreads';
+
+describe('components/channel_list/categories/body', () => {
+    let database: Database;
+
+    beforeAll(async () => {
+        const server = await TestHelper.setupServerDatabase();
+        database = server.database;
+    });
+
+    it('render without error', () => {
+        const wrapper = renderWithEverything(
+            <UnreadsCategory unreadChannels={[]}/>,
```

Comment:
this test seems to be doing nothing, so I wonder if we even need it

Reply:
It's the same with the `channel_list/categories/categories.test.tsx` - ideally our data would be static (vs generated; see https://jestjs.io/docs/snapshot-testing#2-tests-should-be-deterministic) but until then all we can test is that it renders without errors.